Боярников Семён

StarLordJ opened the pull request.

StarLordJ committed:
Опять опаздываю((Больше не повторится!

sinseveriya commented:
@StarLordJ тесты не проходят ([тесты](https://travis-ci.org/urfu-2015/verstka-tasks-2/builds/85407754)) из-за того, что есть табуляции и запрещенные к применению элементы или атрибуты. Поправь, пожалуйста

sinseveriya commented on index.html:
```diff
@@ -1,3 +1,389 @@
+    <body>
+        <h1>Бабуленькин сервант</h1>
+        <section itemscope itemtype="http://schema.org/Book">
+    	    <table width="700px" height="600" cellpadding="5px" cellspacing="0">
```
`700px` = 700

sinseveriya replied:
Интересно, что высота указана без `px`, а ширина с `px`

StarLordJ replied:
ну, раз разницы не было, то и убирать px не стал, а там уже копипаст и всё сохранилось)

sinseveriya replied:
Разница в объеме кода. Надо убрать

sinseveriya commented on index.html:
```diff
@@ -1,3 +1,389 @@
+    	<meta http-equiv="Content-Type" content="text/html; charset=windows-1251">
+    </head>
+    <body>
+        <h1>Бабуленькин сервант</h1>
```
`header`

sinseveriya commented on index.html:
```diff
@@ -1,3 +1,389 @@
+    <head>
+        <title>Бабуленькин сервант</title>
+    	<meta charset = "UTF-8">
+    	<meta http-equiv="Content-Type" content="text/html; charset=windows-1251">
```
Зачем?

sinseveriya commented on index.html:
```diff
@@ -1,3 +1,389 @@
+    	            <td><strong>Издательство:</strong> <span itemprop="publisher">АСТ, Астрель</span></td>
+    	        </tr>
+    	        <tr>
+    	    	    <td><strong>ISBN:</strong> <span itemprop="isbn">5-17-006400-4,978-5-17-006400-7</span>
```
`abbr`

sinseveriya commented on index.html:
```diff
@@ -1,3 +1,389 @@
+                    </td>
+    	        </tr>
+    	        <tr>
+    	            <td><strong>Год:</strong> <span intemprop="datePublished">1867</span></td>
```
Пробелы между тегами?

sinseveriya commented on index.html:
```diff
@@ -1,3 +1,389 @@
+    	        </tr>
+    	        <tr>
+    	    	    <td><strong>Жанр:</strong> 
+                        <span itemprop="genre">Русская классика</span>, 
```
Это два отдельных жанра разве?

sinseveriya commented on index.html:
```diff
@@ -1,3 +1,389 @@
+    	    	    <td><strong>Переплет:</strong> <span itemprop="binding">Твердый переплет</span></td>
+    	        </tr>
+    	        <tr>
+    	    	    <td><strong>Дата размещения:</strong> <span itemprop="dateCreated">8-10-2015</span>
```
Это унести в элемент `time`,  а для людей написать в человекочитаемом формате

sinseveriya commented on index.html:
```diff
@@ -1,3 +1,389 @@
+    	        </tr>
+    	        <tr>
+    	            <td colspan="2"><strong>Описание:</strong>
+                        <p><span itemprop="description">Великий роман-эпопея Льва Толстого
```
`span` не нужен

sinseveriya commented on index.html:
```diff
@@ -1,3 +1,389 @@
+                </tr>
+                <tr>
+                    <td colspan="2"><strong>Описание:</strong>
+                        <p><span itemprop="description">Harry Potter has no idea how famous he is. That's
```
Атрибуты `lang` и `translate`

sinseveriya commented on index.html:
```diff
@@ -1,3 +1,389 @@
+            </table>
+        </section>
+        <section itemscope itemtype="http://schema.org/Person">
+            <table width="700px" height="400" cellpadding="5px" cellspacing="0">
```
Без `px`

sinseveriya commented on index.html:
```diff
@@ -1,3 +1,389 @@
+                        <span itemprop="email">
+                            <a href="mailto:info@jkrowlingpr.com">info@jkrowlingpr.com</a>
+                        </span></td>
+                </tr>
```
В информации еще есть сайт

sinseveriya commented on index.html:
```diff
@@ -1,3 +1,389 @@
+                                шикарный лайнер столкнется с айсбергом в холодных водах Северной Атлантики,
+                                и их страстная любовь превратится в схватку со смертью.
+                           </span></p>
+                        <iframe width="680" height="420" src="https://www.youtube.com/embed/zCy5WQ9S4c0" frameborder="0" allowfullscreen>
```
`iframe` ок, но хотелось бы посмотреть на использование `video`

sinseveriya commented:
@StarLordJ Семён, твой код не соответствует [кодстайлу](https://github.com/urfu-2015/guides). Читать очень тяжко. Приведи код в порядок.

StarLordJ committed:
правки

пробелы типа "...>Место рождения:</strong> <span itemprop=...." между тегами нужны, чтобы после двоеточия был отступ

StarLordJ committed:
убрал табуляцию

>< не вкурил, как проверять html документ

sinseveriya commented:
Когда пушишь правки, пиши комментарий с моим ником, чтобы мне пришло уведомление

sinseveriya commented on index.html:
```diff
@@ -1,3 +1,530 @@
+        <section itemscope itemtype="http://schema.org/Book">
+            <table width="700" height="600" cellpadding="5" cellspacing="0">
+            <tr>
+            <td colspan="2">
```
Все еще с форматирование проблемы

sinseveriya commented on index.html:
```diff
@@ -1,3 +1,530 @@
+                </tr>
+                <tr>
+                    <td>
+                        <strong>Язык:</strong> <span itemprop="inLanguage">Русский</span>
```
`strong` тег для логического выделения, а не для стилистического

StarLordJ committed:
ФОрматирование исправил, тег убрал))

sinseveriya

sinseveriya commented:
Принято
